Model/EasyshipCarrier.php

```diff
@@ -207,15 +208,16 @@ protected function _createEasyShipRequest(RateRequest $request)
                 }
 
                 $itemQty = (int)$item->getQty();
+                $unitPriceInclTaxAndDiscount = ($item->getRowTotal() - $item->getDiscountAmount() + $item->getTaxAmount()) / $itemQty;
                 for ($i = 0; $i < $itemQty; $i++) {
                     $items[] = [
                         'actual_weight' => $this->getWeight($item->getProduct()),
                         'height' => $this->getEasyshipHeight($item->getProduct()),
                         'width' => $this->getEasyshipWidth($item->getProduct()),
                         'length' => $this->getEasyshipLength($item->getProduct()),
                         'category' => $this->getEasyshipCategory($item->getProduct()),
-                        'declared_currency' => $currencyCode,
-                        'declared_customs_value' => (float)$item->getPrice(),
+                        'declared_currency' => $currentCurrencyCode,
+                        'declared_customs_value' => $unitPriceInclTaxAndDiscount,
```

Comment:
interesting! not sure why need to deduct discount ~

take shopify as an example, 

https://shopify.dev/api/admin-rest/2022-01/resources/carrierservice

Shopify do have discount feature, when we get request from Shopify, Shopify will not include discount ~
(I remember some client complain before, why free shipping not working...blablabla)

Reply:
ok confirmed by PM so let's deduct~

Reply:
The logic of calculating the item price that sent from platform will be different after this.
Notes:
https://easyship-team.atlassian.net/wiki/spaces/TECH/pages/487325723/Int+Magento+Integration
https://easyship-team.atlassian.net/wiki/spaces/TECH/pages/558465419/Shopify+Rates+at+Checkout+R+C